src/default_options.js

```diff
@@ -0,0 +1,191 @@
+import { replaceRange } from "./changes";
+import { initScrollbars, setGuttersForLineNumbers, themeChanged, updateGutters } from "./cm_utils";
+import { addClass, rmClass } from "./util/dom";
+import { off, on } from "./util/event";
+import { onBlur } from "./display/focus";
+import { getKeyMap } from "./keymap";
+import { defaultSpecialCharPlaceholder } from "./line/line_data";
+import { alignHorizontally } from "./display/line_numbers";
+import Pos from "./Pos";
+import { clearCaches, compensateForHScroll, estimateLineHeights } from "./measurement/position_measurement";
+import { updateScrollbars } from "./display/scrollbars";
+import { updateSelection } from "./display/selection_draw";
+import { mobile, windows } from "./util/browser";
+import { findMaxLine } from "./line/spans";
+import { loadMode, resetModeState } from "./state";
+import { regChange } from "./display/view_tracking";
+
+export var Init = {toString: function(){return "CodeMirror.Init";}};
+
+export function setDefaultOptions(CodeMirror) {
```

Comment:
Actually, I didn't know how to just require with rollup. Also has the nice advantage of not depending on CodeMirror.js.